### ISSUE TITLE
Generate your Rust code by running on windows wont work

### ISSUE THREAD

**Issue by Revnixcad** [houseabsolute/tailwindcss-to-rust#2]
I'm using the tooling in combination with Dioxus
But with step 6 generating rust code:
```
tailwindcss-to-rust \
     --tailwind-config tailwind.config.js \
     --input tailwind.css \
     --output src/css/generated.rs \
     --rustfmt
```
I get the following error message:
```
thread 'main' panicked at 'Could not run tailwindcss: program not found', C:\Users\<user>\.cargo\registry\src\github.com-1ecc6299db9ec823\tailwindcss-to-rust-0.1.3\src\main.rs:196:29
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Maybe i'm doing someting wrong?

I'm using PowerShell on W11

**Comment by autarch** [comment]
Do you have the `tailwindcss` program installed, and is it in your `PATH`? You can get a binary version of it from https://github.com/tailwindlabs/tailwindcss/releases

**Comment by Revnixcad** [comment]
yes all installed by the instructions and works fine  `npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch` does what it must do.
I have to mention that plain `tailwindcss` is not working in the terminal.

**Comment by autarch** [comment]
The tool is trying to just run `tailwindcss` directly. If that's not in your path it won't work. Can you try installing a binary release like I linked above?

**Comment by autarch** [comment]
Or alternatively, you could put a shim in your path that executes `npx tailwindcss $@`. That might work too.

**Comment by Revnixcad** [comment]
> Or alternatively, you could put a shim in your path that executes `npx tailwindcss $@`. That might work too.

i feel like a fool right know
can't figure it out how to set this up in W11

**Comment by autarch** [comment]
I don't really know much about shell scripting on Windows either, I'm afraid. But you should be able to install the `tailwindcss` binary on Windows easily enough.

**Comment by Revnixcad** [comment]
i figure it out, i set `C:\Users\<username>\node_modules\.bin` to my windows system path variables and `tailwindcss` seems to work right know. 

know i wil test step 6 to to generating rust code will let you know shortly

**Comment by autarch** [comment]
Ah, that makes sense as a fix. I will update the docs to mention the issue with having `tailwindcss` in your path.

**Comment by Revnixcad** [comment]
still get the same error 
while command `tailwindcss` give  following respons in my terminal:

```
tailwindcss v3.0.24

Usage:
   tailwindcss [--input input.css] [--output output.css] [--watch] [options...]
   tailwindcss init [--full] [--postcss] [--types] [options...]

Commands:
   init [options]

Options:
   -i, --input              Input file
   -o, --output             Output file
   -w, --watch              Watch for changes and rebuild as needed
       --content            Content paths to use for removing unused classes
       --postcss            Load custom PostCSS configuration
   -m, --minify             Minify the output
   -c, --config             Path to a custom config file
       --no-autoprefixer    Disable autoprefixer
   -h, --help               Display usage information
```

**Comment by autarch** [comment]
I don't know what to tell you other than that `tailwindcss` is not in your path when you execute the `tailwindcss-to-rust` executable. I don't know enough about Windows path handling to figure out what the problem is beyond that.

**Comment by Revnixcad** [comment]
i will try to investigate further, will let you know...

**Comment by autarch** [comment]
I just released a new version of the generator, v0.1.4. This adds a new `--tailwindcss` CLI argument which you can use to specify the path to the `tailwindcss` executable. Using that might fix the problem you're having.

**Comment by Revnixcad** [comment]
awesome i got it working right know. 

1. downloaded [tailwindcss-windows-x64.exe](https://github.com/tailwindlabs/tailwindcss/releases/download/v3.0.24/tailwindcss-windows-x64.exe) in default download folder
2. run the following command `tailwindcss-to-rust --tailwind-config tailwind.config.js --tailwindcss "C:\Users\<username>\Downloads\tailwindcss-windows-x64.exe" --input src/input.css --output src/css/generated.rs --rustfmt`

It works!

With this knowledge i tried also the following, to work with the default.
1. downloaded [tailwindcss-windows-x64.exe](https://github.com/tailwindlabs/tailwindcss/releases/download/v3.0.24/tailwindcss-windows-x64.exe) in default download folder
2. copy tailwindcss-windows-x64.exe to `C:\Program Files\Tailwindcss\tailwindcss-windows-x64.exe`
3. rename `tailwindcss-windows-x64.exe` to `tailwindcss.exe` 
4. add `C:\Program Files\Tailwindcss` to windows system variables
5. run `refreshenv` to opdate env in terminal
6. command `tailwindcss` is working fine at this point and `tailwindcss-to-rust --tailwind-config tailwind.config.js --tailwindcss "C:\Program Files\Tailwindcss\tailwindcss"  --input src/input.css --output src/css/generated.rs --rustfmt` configuration works fine.
7. But `tailwindcss-to-rust --tailwind-config tailwind.config.js  --input src/input.css --output src/css/generated.rs --rustfmt` will still not work.. :(


But thx for the fix
